assets/stylesheets/common/chat-toggle-membership-btn.scss

```diff
@@ -0,0 +1,9 @@
+.chat-toggle-membership-btn {
```

Comment:
I am not sure if this is even needed because of `joinClasses` and `leaveClasses`